metaci/build/models.py

```diff
@@ -686,6 +682,15 @@ def run_flow(self, project_config, org_config):
         # Run the flow
         return self.flow_instance.run(org_config)
 
+    def set_release_dates(self, options):
+        if self.build.plan.role == "release" and self.build.release:
+            options["github_release_notes"] = {
+                "sandbox_date": self.build.release.sandbox_push_date,
+                "production_date": self.build.release.production_push_date,
+                "trial_info": self.build.release.trialforce_id,
```

Comment:
Still doesn't make sense to include the trial_info (because it isn't known when uploading the release). Please remove it.